content/en/blog/2022/k8sattributes.md

```diff
@@ -0,0 +1,261 @@
+---
+title: Improved troubleshooting using k8s attributes
+linkTitle: Kubernetes attributes
+date: 2022-06-27
+author: Ruben Vargas
+spelling:
+  cSpell:ignore k8sattributes k8sattributesprocessor K8sattributes k8sprocessor
+  cSpell:ignore K8sprocessor KUBE
+---
+
+Attaching Kubernetes resource metadata to OpenTelemetry traces can be very
+useful because it lets you identify which resource (such as a pod) is failing or
+having performance problems. It is also generally useful for correlating across
+other signals.
+
+In this article, you'll learn how to configure the OpenTelemetry collector to
+use the `k8sattributesprocessor` processor in different scenarios. At the end,
+we present some alternatives.
```

Comment:
> At the end, we present some alternatives.

Really?

Reply:
May be is just a short mention, but we present an alternative of using the resource detector processor.

Reply:
I'd propose dropping that sentence.